packages/itwinui-react/src/core/ExpandableBlock/ExpandableBlock.tsx

```diff
@@ -120,43 +182,150 @@ export const ExpandableBlock = React.forwardRef((props, ref) => {
+          <ExpandableBlock.Title>{label}</ExpandableBlock.Title>
+        </ExpandableBlock.LabelArea>
+      )}
+    </Box>
+  );
+}) as PolymorphicForwardRefComponent<'div', ExpandableBlockHeaderOwnProps>;
+ExpandableBlockHeader.displayName = 'ExpandableBlock.Header';
+
+// ----------------------------------------------------------------------------
+// ExpandableBlock.LabelArea component
+
+const ExpandableBlockLabelArea = polymorphic.span('iui-expandable-block-label');
+ExpandableBlockLabelArea.displayName = 'ExpandableBlock.LabelArea';
+
+// ----------------------------------------------------------------------------
+// ExpandableBlock.Title component
+
+const ExpandableBlockTitle = React.forwardRef((props, forwardedRef) => {
+  const { className, children, ...rest } = props;
+  return (
+    <Box className={cx('iui-title', className)} ref={forwardedRef} {...rest}>
+      {children}
+    </Box>
+  );
+}) as PolymorphicForwardRefComponent<'div'>;
+ExpandableBlockTitle.displayName = 'ExpandableBlock.Title';
+
+// ----------------------------------------------------------------------------
+// ExpandableBlock.Caption component
+
+const ExpandableBlockCaption = polymorphic('iui-caption');
+ExpandableBlockCaption.displayName = 'ExpandableBlock.Caption';
+
+// ----------------------------------------------------------------------------
+// ExpandableBlock.EndIcon component
+
+const ExpandableBlockEndIcon = React.forwardRef((props, forwardedRef) => {
+  const { className, children, ...rest } = props;
+  const { status } = useSafeContext(ExpandableBlockContext);
+
+  const icon = children ?? (status && StatusIconMap[status]());
+  return (
+    <Icon className={className} fill={status} ref={forwardedRef} {...rest}>
+      {icon}
+    </Icon>
+  );
+}) as PolymorphicForwardRefComponent<
+  'span',
+  React.ComponentPropsWithoutRef<typeof Icon>
+>;
+ExpandableBlockEndIcon.displayName = 'ExpandableBlock.EndIcon';
+
+// ----------------------------------------------------------------------------
+// ExpandableBlock.Content component
+
+const ExpandableBlockContent = React.forwardRef((props, forwardedRef) => {
+  const { className, children, ...rest } = props;
+  const { isExpanded } = useSafeContext(ExpandableBlockContext);
+
+  return (
+    <WithCSSTransition in={isExpanded}>
       <Box
-        role='button'
-        aria-expanded={expanded}
-        className='iui-header'
-        aria-disabled={disabled}
-        tabIndex={0}
-        onClick={handleToggle}
-        onKeyDown={onKeyDown}
+        className={cx('iui-expandable-content', className)}
+        ref={forwardedRef}
+        {...rest}
       >
-        <SvgChevronRight className='iui-icon' aria-hidden />
-        <Box as='span' className='iui-expandable-block-label'>
-          <Box className='iui-title'>{title}</Box>
-          {caption && <Box className='iui-caption'>{caption}</Box>}
-        </Box>
-        {icon && <Icon fill={status}>{icon}</Icon>}
+        <Box>{children}</Box>
```

Comment:
Why do you have this additional Box for children?

Reply:
Because of this in the css file:
![image](https://github.com/iTwin/iTwinUI/assets/22462084/b43463ad-44b6-4f98-95d9-9450dd3ad577)